BBCSounds/BBCSoundsFeeder.pm

```diff
@@ -31,6 +32,7 @@ use POSIX qw(strftime);
 use HTTP::Date;
 use Digest::MD5 qw(md5_hex);
 use Slim::Utils::Strings qw(string cstring);
+use XML::Simple qw(:strict);
```

Comment:
This import seems to break a lot of stuff in LMS !

Reply:
Eeek!

It turns out that this is a [bug in XML::Simple that was fixed a long time ago](https://rt.cpan.org/Public/Bug/Display.html?id=41562), but LMS still bundles a version from 2006.

I'll come up with a version of this patch that doesn't use `:strict`.

Reply:
Resubmitted as https://github.com/expectingtofly/LMS_BBC_Sounds_Plugin/pull/86